### PR TITLE
Dev6.0

### PR DESCRIPTION
**Pull request by devu4**
Ignore menu commits, done cowboy. 

Could you just check it over. Am I alright to use this format for all of the eventdays? 
Also should I strip weapons onavailableLR?

### CODE REVIEW

**Comment by shanapu** [comment]
Thanks mate for the good suggestion and the fast implementation! I realy appreciate your help!

I'm bit busy next days, so i can't check all days for myself or go deeper in the code.

Before I merge your PR please undo Commit `Menu changes for gfl` https://github.com/shanapu/MyJailbreak/pull/68/commits/ca36988639c72d102785db12677205973d0ae75d

File: addons/sourcemod/scripting/MyJailbreak/duckhunt.sp [hunk 1]
```diff
@@ -59,6 +61,8 @@ char huntermodel[256] = "models/player/custom_player/legacy/tm_phoenix_heavy.mdl
 char g_sCustomCommand[64];
 char g_sEventsLogFile[PLATFORM_MAX_PATH];
 char g_sAdminFlag[32];
+char g_sModelPathCT[256];
```
Comment (shanapu) [inline addons/sourcemod/scripting/MyJailbreak/duckhunt.sp]: char g_sModelPathCT[MAXPLAYERS+1][256];
char g_sModelPathT[MAXPLAYERS+1][256];

to store different playermodels to each client

GetEntPropString(client, Prop_Data, "m_ModelName", g_sModelPathCT[client], sizeof(g_sModelPathCT[client]));
GetEntPropString(client, Prop_Data, "m_ModelName", g_sModelPathT[client], sizeof(g_sModelPathT[client]));

SetEntityModel(client, g_sModelPathCT[client]);
SetEntityModel(client, g_sModelPathT[client]);

so every player get **his** old model and not the model of the last looped client

**Comment by devu4** [comment]
All done, did own client skin on zombie day too. Also removed menu commits

File: addons/sourcemod/translations/MyJailbreak.War.phrases.txt [hunk 2]
```diff
@@ -161,11 +161,11 @@
 	"war_rounds"
 	{
 		"#format"   "{1:i}"
```
Comment (shanapu) [inline addons/sourcemod/translations/MyJailbreak.War.phrases.txt]: need to be

`"#format"   "{1:i},{2:i}"`

**Comment by shanapu** [comment]
im not sure if we should bool the roundend too.

e.g.

`//Round End

public void RoundEnd(Handle event, char[] name, bool dontBroadcast)
{
	int winner = GetEventInt(event, "winner");
	
	if (IsCowBoy)
	{
		//code
		
		if (g_iRound == g_iMaxRound && !gc_bAllowLR.BoolValue)
		{
			//code
		}
	}
	//code
}
`

**Comment by shanapu** [comment]
as we discussed, merged and done the commits by myself.

Thx for your awseome work!
Thx for be a part of MyJailbreak!

**Pull request merged**